businessRules/ideas.js

```diff
@@ -19,34 +22,22 @@ function sortedByDonationSumIdeas() {
     return sortedIdeasList;
 }
 
-function createIdeas(header, description, askedSum, userId) {
-    if (!header) {
-        throw new Error('Error. Please, check your header.');
-    }
+function createIdea(header, description, askedSum, userId) {
+    validateHeader(header);
 
-    if (!description) {
-        throw new Error('Error. Please, check your description.');
-    }
+    validateDescription(description);
 
-    if (!askedSum) {
-        throw new Error('Error. Please, check your asked sum.');
-    }
+    validateAskedSum(askedSum);
 
-    if (!userId) {
-        throw new Error('Error. Please, check if you are logged in.');
-    }
+    validateUserId(userId);
 
-    createIdea(header, description, askedSum, userId);
+    create(header, description, askedSum, userId);
```

Comment:
The `create` name is too abstract. Can you improve it?

Reply:
Renamed